examples/basic.rs

```diff
@@ -13,12 +13,12 @@ async fn main() -> Result<()> {
     let session = Session::connect(uri, None).await?;
     session.refresh_topology().await?;
 
-    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &scylla::values!()).await?;
```

Comment:
it's a little bit out of scope for this patch, but I think we need a separate wrapper which allows to send a query without this boilerplate. Something like `session.query_simple("string")`, so people aren't forced to use these weird empty values every single time.

I still think it makes sense to add it in this series, because examples like
```rust
    session
        .query(
            "CREATE TABLE IF NOT EXISTS ks.t2 (a int, b int, c text, primary key (a, b))",
            &scylla::values!(),
        )
```
become way too complicated and weird, as opposed to
```rust
session.query_simple("CREATE TABLE IF NOT EXISTS ks.t2 (a int, b int, c text, primary key (a, b))")
```